src/components/editor/EditorArticleController.jsx

```diff
@@ -59,6 +63,16 @@ export default class EditorArticleController extends FalcorController {
     return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
   }
 
+  componentWillReceiveProps(nextProps) {
+    super.componentWillReceiveProps(nextProps);
+    this.safeSetState({
+      changed: "none",
```

Comment:
At the moment it is not binary no. Possible values are: `['none' ,'changed', 'saving', 'saved']`